presto-raptor/src/main/java/com/facebook/presto/raptor/storage/StorageManagerConfig.java

```diff
@@ -63,6 +63,7 @@
     private int recoveryThreads = 10;
     private int organizationThreads = 5;
     private boolean organizationEnabled = true;
+    private boolean tableSupportsDeltaDelete;
```

Comment:
No need to have this config. Default the table property value also to false.

Reply:
According to the early discussion, we want to have the ability to control the default behavior of all the table newly created, instead of making it a manual operation.

Besides, this feature is a good to have, then maybe we don't need to remove it